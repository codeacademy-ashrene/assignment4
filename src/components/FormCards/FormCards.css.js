import { StyleSheet } from 'react-native';

export default StyleSheet.create({

  card: {
    // width: '60%',
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'center',
    margin: 20,
  },
  cardTitle: {
    flexGrow: 1,
    justifyContent: 'flex-start',
    backgroundColor: '#FFFFFF',
    height: 100,
    width: 300,
    padding: '5%',
  },
  cardTitleText: {
    color: '#000000',
    fontSize: 22,
  },
  hr: {
    flexGrow: 1,
    borderBottomColor: '#000000',
    borderBottomWidth: 1,
  },
  cardBottom: {
    flexGrow: 1,
    backgroundColor: '#FFFFFF',
    height: 50,
    width: 300,
  },
  cardBottomText: {
    color: '#000000',
    fontSize: 18,
    padding: '3%',
  },
});
