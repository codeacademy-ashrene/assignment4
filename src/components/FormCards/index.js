import React, { Component } from 'react';
import { View, Text } from 'react-native';
import styles from './FormCards.css';

class Header extends Component {
  render() {
    return (
      <View style={styles.card}>
        <View style={styles.cardTitle}>
          <Text style={styles.cardTitleText}>Code Academy Feedback 2019</Text>
        </View>
        <View style={styles.hr} />
        <View style={styles.cardBottom}>
          <Text style={styles.cardBottomText}>Created On: 26/02/2019</Text>
        </View>
      </View>
    );
  }
}

export default Header;
