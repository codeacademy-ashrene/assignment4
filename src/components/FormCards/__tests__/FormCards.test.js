import React from 'react';
import renderer from 'react-test-renderer';
import FormCards from '..';


describe('FormCards', () => {
  it('should render FormCards', () => {
    const tree = renderer.create(<FormCards />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
