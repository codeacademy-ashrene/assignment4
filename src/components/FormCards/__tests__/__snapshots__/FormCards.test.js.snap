// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCards should render FormCards 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "margin": 20,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flexGrow": 1,
        "height": 100,
        "justifyContent": "flex-start",
        "padding": "5%",
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 22,
        }
      }
    >
      Code Academy Feedback 2019
    </Text>
  </View>
  <View
    style={
      Object {
        "borderBottomColor": "#000000",
        "borderBottomWidth": 1,
        "flexGrow": 1,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flexGrow": 1,
        "height": 50,
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 18,
          "padding": "3%",
        }
      }
    >
      Created On: 26/02/2019
    </Text>
  </View>
</View>
`;
